bin/build_flang-legacy.sh

```diff
@@ -55,6 +55,7 @@ LLVMCMAKEOPTS="\
 -DLLVM_INCLUDE_EXAMPLES=0 \
 -DLLVM_INCLUDE_TESTS=0 \
 -DLLVM_INCLUDE_DOCS=0 \
+$_cxx_flag \
```

Comment:
is this correct ?   $_cxx_flag

Reply:
Yes see lines 34-38 above, which will set _cxx_flag for rhel-7/8
 _cxx_flag="-DCMAKE_CXX_FLAGS='-D_GLIBCXX_USE_CXX11_ABI=0'"